docs/src/clusters.md

```diff
@@ -36,15 +36,15 @@ solana config set --url https://devnet.solana.com
 
 ```bash
```

Comment:
The doc/ and `bank.rs` changes in here look just fine, why don't you just land those as a separate PR while we work through the `ci/` files in this PR

Reply:
There is no strong reason to create separate PRs. I just thought it's not worth to be its own prs. `bank.rs` changes are needed to trigger `live-cluster` tests. (yeah, I could improve the `ci/buildkite-pipeline.sh`). And `docs` chagnes somewhat mentions this pr about the update notice. So separating them introduces a bit of work.